src/mprisplayer.cpp

```diff
@@ -93,21 +93,38 @@ qlonglong MprisPlayerPrivate::position() const
 
 void MprisPlayerPrivate::setLoopStatus(const QString &value)
 {
-    bool ok;
-    int enumVal = QMetaEnum::fromType<Mpris::LoopStatus>().keyToValue(value.toUtf8(), &ok);
+    int enumVal;
+    bool ok = true;
```

Comment:
I would prefer to skip the `ok`, and just `sendErrorReply(); return`  in else, but either way works.

Reply:
Both having good sides, this one avoiding return in the middle.